test-data/unit/check-isinstance.test

```diff
@@ -1443,6 +1443,28 @@ else:
 [builtins fixtures/isinstancelist.pyi]
 
 
+[case testIssubclasDestructuringUnions]
+from typing import Union, List, Tuple, Dict, Type
+def f(x: Union[Type[int], Type[str], Type[List]]) -> None:
```

Comment:
Done! 

Interesting, when I `git reset --hard @~` and `git push -f`, I effectively remove the last commit from this PR. Then AppVeyor decided to rebuild, while Travis decided not to. At first I thought that AppVeyor was smarter because it noticed that master branch changed so it figured the new merge with master will be different, and it's worth rebuilding.

It turns out that AppVeyor rebuilds regardless of whether master changed since the last time it built the commit; in fact, it even rebuilt on my own branch in the forked repo.

So, when they already built for the commit that just became the head of the PR or branch:

- AppVeyor always rebuilds, even if there's absolutely no need
- Travis does not rebuild, even if it's arguably worth rebuilding due to the change in master